### ISSUE TITLE
CX Priority Selection

### ISSUE THREAD

**Issue by Gladi** [Jacudibu/PRUNner#1]
https://github.com/Jacudibu/PRUNner/projects/1#card-62816375

Where would  this CX Selection be?
There is no selection on the planner currently.

**Comment by Jacudibu** [comment]
Yeah, right now you'll need to edit the UserData/empire.json file manually – right at the top there should be this field:
```
  "GlobalSettings": {
    "PriceDataPreferenceOrder": [
      "PlanetOverrides",
      "EmpireOverrides",
      "NC1Bid",
      "NC1Average",
      "MMBuy",
      "NC1Ask",
      "MMSell"
    ]
  },
```
Just change the order or change NC1 with any other CX and then reload your Data :)

**Comment by Jacudibu** [comment]
Implemented in v0.4
